antropy/entropy.py

```diff
@@ -706,6 +706,15 @@ def _lz_complexity(binary_string):
     return complexity
 
 
+@np.vectorize
```

Comment:
As I said in the PR comment: we need this so that we can apply the function to a numpy array. Otherwise, the `x == 0` would become a boolean array, and then the `if x == 0` would trip over because 'the truth value of an array with more than one element is ambiguous'.